### DISCUSSION TITLE
Windows keyboard not responding

### DISCUSSION THREAD

**Discussion by nwfungi** [soundshed/soundshed-app#43] [Q&A]
Keyboard input VERY slow or not responding at all in search box after newest update

**Comment by webprofusion-chrisc** [comment]
Thanks for raising the issue. I'm guessing our new keyboard mappings are messing with stuff (you can now press 1-4 to change channels on the amp).

**Reply by nwfungi** [reply]
Just tested, 1-4 works as should, no other keyboard input works in any text requester

**Reply by webprofusion-chrisc** [reply]
OK, fixed in new version.

**Comment by nwfungi** [comment]
Sweet!  Ill kick the tires today and see how it purrs :)

On 8/5/2021 4:55:01 AM, Christopher Cook ***@***.***> wrote:
OK, fixed in new version.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub [https://github.com/soundshed/soundshed-app/discussions/43#discussioncomment-1134356], or unsubscribe [https://github.com/notifications/unsubscribe-auth/AECQLA4ECYZANIDZ6F2XSSDT3J3ZJANCNFSM5BSGJ6MQ].
Triage notifications on the go with GitHub Mobile for iOS [https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675] or Android [https://play.google.com/store/apps/details?id=com.github.android&amp;utm_campaign=notification-email].
[c6ed2e37-9b38-4b9b-a0ce-1f0b7168baa8]

**Reply by nwfungi** [reply]
Looking good! Nice work!
